python/tvm/relay/backend/contrib/tidl.py

```diff
@@ -0,0 +1,1407 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""TIDL backend compiler"""

```

Comment:
Can we add a __init__.py file to contrib directory? without that git does not clone this file correctly

Reply:
Could you elaborate a bit more? I didn't get why we need `init.py` for git?

Reply:
Oh I was looking for \__init\__.py file required to make Python treat directories containing the file as packages. When I git clone, this file (python/tvm/relay/backend/contrib/tidl.py) does not get cloned and I think its because this directory (python/tvm/relay/backend/contrib) is missing \__init\__.py file.

Reply:
__init__.py was added.